src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public interface Figure {
+    String draw();
+}
+
+
+
+
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present